internal/cmd/agent/deployer/applied/diff.go

```diff
@@ -81,9 +82,18 @@ func Diff(plan apply.Plan, bd *fleet.BundleDeployment, ns string, objs ...runtim
 	return plan, nil
 }
 
-func normalizers(live objectset.ObjectByGVK, bd *fleet.BundleDeployment) (diff.Normalizer, error) {
+// newNormalizers creates a normalizer that removes fields from resources.
+// The normalizer is composed of:
+//
+//   - StatusNormalizer
+//   - MutatingWebhookNormalizer
+//   - ValidatingWebhookNormalizer
```

Comment:
Where do those 3 come from?

Reply:
Prepended by the fleet normalizer package when calling `normalizers.New` in the end:
https://github.com/rancher/fleet/blob/main/internal/cmd/agent/deployer/normalizers/norm.go#L22-L39